src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -191,15 +193,18 @@ private void ShowGoldBar()
             Debug.Assert(_experimentationService.IsExperimentEnabled(InternalFlightName) ||
                          _experimentationService.IsExperimentEnabled(ExternalFlightName));
 
-            string message = ServicesVSResources.Disabling_the_extension_0_unbound_your_keyboard_bindings;
+            string message = ServicesVSResources.We_notice_you_suspended_0_Reset_keymappings_to_continue_to_navigate_and_refactor;
             KeybindingsResetLogger.Log("InfoBarShown");
             var infoBarService = _workspace.Services.GetRequiredService<IInfoBarService>();
             infoBarService.ShowInfoBarInGlobalView(
                 string.Format(message, ReSharperExtensionName),
-                new InfoBarUI(title: ServicesVSResources.Restore_Visual_Studio_keybindings,
+                new InfoBarUI(title: ServicesVSResources.Reset_Visual_Studio_default_keymapping,
                               kind: InfoBarUI.UIKind.Button,
                               action: RestoreVsKeybindings),
-                new InfoBarUI(title: ServicesVSResources.Use_Keybindings_for_extensions,
+                new InfoBarUI(title: string.Format(ServicesVSResources.Apply_0_keymapping_scheme, ReSharperKeyboardMappingName),
+                              kind: InfoBarUI.UIKind.Button,
+                              action: OpenExtensionsHyperlink),
```

Comment:
same action for this apply ReSharperKeyboardMapping and VSCodeKeyboardMapping？

Reply:
Yes, for now they will both go to the same FWLink and each button might later be updated to make the specific change for the user.